### ISSUE TITLE
Build failure for 0.2

### ISSUE THREAD

**Issue by snoyberg** [ekmett/lens-action#5]
I just ran into this, which looks to be a regression for #3:

```
Unpacking to lens-action-0.2/
Resolving dependencies...
[1 of 1] Compiling Main             ( dist/setup/setup.hs, dist/setup/Main.o )

dist/setup/setup.hs:8:1: Warning:
    The import of ‘Package’
    from module ‘Distribution.Package’ is redundant

dist/setup/setup.hs:11:1: Warning:
    The import of ‘copyFiles’
    from module ‘Distribution.Simple.Utils’ is redundant

dist/setup/setup.hs:13:1: Warning:
    The import of ‘HaddockFlags, Flag, haddockDistPref’
    from module ‘Distribution.Simple.Setup’ is redundant

dist/setup/setup.hs:15:1: Warning:
    The import of ‘Distribution.Text’ is redundant
      except perhaps to import instances from ‘Distribution.Text’
    To import instances alone, use: import Distribution.Text()

dist/setup/setup.hs:16:1: Warning:
    The import of ‘normal’
    from module ‘Distribution.Verbosity’ is redundant
Linking ./dist/setup/setup ...
Configuring lens-action-0.2...
Building lens-action-0.2...
Preprocessing library lens-action-0.2...

src/Control/Lens/Action.hs:64:8:
    Could not find module ‘Control.Lens.Action.Type’
    Use -v to see a list of the files searched for.
```

**Comment by jonsterling** [comment]
One lesson I have learnt as a maintainer of packages is to always try and build the package directly from the tarball generated by `cabal sdist` before uploading; this can catch some problems that do not come up during a normal `cabal build` or `cabal check`. (I haven't looked into this so I don't know if it is as simple as some module missing from the `.cabal` file, but it sounds plausible.)

**Comment by snoyberg** [comment]
@hvr's [Travis template](https://github.com/hvr/multi-ghc-travis#travisyml-template) does this, and I highly recommend it

**Comment by peti** [comment]
A fix for this issue would be very much appreciated because this build failure breaks the users of this package, too: http://packdeps.haskellers.com/reverse/lens-action. It appears that the fix is relatively simple, no?

**Comment by jonsterling** [comment]
In case the maintainers are overwhelmed, I'd be happy to help maintain it, seeing as I appear to be one of the few people who actually likes using it :)

**Comment by glguy** [comment]
Resolved again in 5131ab7d9485fe938cb21742461f4f6525fe7e0b
